added uuid of admin group to monitor by list

eschondorf opened the pull request.

eschondorf committed:
added uuid of admin group to monitor by list

eschondorf committed:
added uuid of admin group to monitor by list

BenGalewsky commented on aws/automate_manager.py:
```diff
@@ -126,7 +126,7 @@ def submit(self, mdf_rec, organization,
         }
         print("Flow is ", self.flow)
-        flow_run = self.flow.run_flow(automate_rec, monitor_by=[submitting_user_id])
+        flow_run = self.flow.run_flow(automate_rec, monitor_by=[submitting_user_id, "5fc63928-3752-11e8-9c6f-0e00fd09bf20"])
```
Would be better not to have an MDF specific magic uuid here in this general purpose class. Perhaps make a new parameter to `submit` method that has the list of monitor_by values. We could move this magic UUID up into submit_dataset.

Make sure the magic uuid is well documented there

eschondorf replied:
Just made a commit with the update.

eschondorf committed:
Updating so that magic uuid is in submit dataset not automate manager

eschondorf committed:
Updating so that magic uuid is in submit dataset not automate manager

BenGalewsky committed:
Expand CI job to test all branches and deploy to develop Gateway (#68)

CI Job tests all branches and pushes development branches to test Lambda function

BenGalewsky committed:
Merge branch 'monitor_by_admin' of https://github.com/materials-data-facility/connect_server into monitor_by_admin

eschondorf committed:
syntax changes

eschondorf committed:
update metadata_only automate manager test

eschondorf committed:
missing comma

eschondorf committed:
adding prefixes
